neqo-http3/src/connection_client.rs

```diff
@@ -2350,8 +2353,9 @@ mod tests {
         let out = server.conn.process(None, now());
         let _out = client.process(out.dgram(), now());
 
-        let header_ready_event = |e| matches!(e, Http3ClientEvent::HeaderReady { .. });
-        assert!(!client.events().any(header_ready_event));
+        // TODO fine a way to send encoded header but do not send instrtuction.
+        // let header_ready_event = |e| matches!(e, Http3ClientEvent::HeaderReady { .. });
+        // assert!(!client.events().any(header_ready_event));
```

Comment:
No, that will not help.
I want reordering. If I limit the stream the header_block_encoding will not use the dynamic table because it needs to send instructions but it cannot.
I want to encode headers that use dynamic table and also add new entries to the table. If I want to add new entries I need to have place in the stream (no flow-control limited). 
I think I need to use neqo-transport connection here.

Reply:
Oh, I could make this work with tricky way of limiting flow-control so that it we can only send table instructions, but not headers. and let that packet be delay. then increase flow-control and send headers. this headers will be blocked until the instruction packet is received.... this is fun.